app/styles/_overview.less

```diff
@@ -97,59 +106,94 @@
   .standalone-service {
     width: 100%;
     @media (min-width: @screen-sm-min) {
-      width: 50%;
-
-      &:nth-child(odd) {
-          padding-right: 5px;
-      }
-
-      &:nth-child(even) {
-          padding-left: 5px;
-      }
-
+      .flex(@columns: 0 0 49%);
+    } 
+    @media (min-width: @screen-md-min) {
+      .flex(@columns: 0 0 49.375%);
     }
-    .service-group-body .overview-services {
-      overview-service {
-        max-width: none;
+    .no-child-services-message {
+      display: none;
+    }
+    overview-service {
+      .flex(@columns: 0 0 100%) !important;
+    }
+    overview-service-group {
+      .flex-display(@display: flex);
+    }
+    .service-group > div {
+      .flex-display(@display: flex);
+      .flex-direction(@direction: column);
+      height: 100%;
+    }
+    .service-group-body {
+      margin-top: 0;
+      .overview-services .primary-service {
+        .deployment-block {
+          .flex-display(@display: flex);
+        }
       }
-      .deployment-tile {
-        margin-top: 0;
+    }
+    .alert-wrapper .alert {
+      margin-bottom: @service-group-margin;
+      margin-top: 0;
+    }
+    .service-group-body {
+      .flex-display(@display: flex);
+      .flex-direction(@direction: column);
+      width: 100%;
+      .overview-services {
+        .flex-direction(@direction: row);
+        @media (min-width: @screen-sm-min) {
+          min-height: 345px;
+        }
+        @media (min-width: @screen-md-min) {
+          min-height: 255px;
+        }
+        overview-service,.no-child-services-message {
+          max-width: none;
+          margin-top: 0;
+        }
       }
     }
   }
+  .overview-monopods {
+    .justify-content(@justify: space-between);
+  }
   .no-service {
+    .flex-display(@display: flex);
     width: 100%;
-    margin-bottom: 10px;
-
-    @media (min-width: @screen-sm-min) {
-      width: 50%;
-
-      &:nth-child(odd) {
-          padding-right: 5px;
-      }
-
-      &:nth-child(even) {
-          padding-left: 5px;
-      }
+    margin-bottom: 20px;
+    position: relative;
+    &:before {
+      content: "";
+      display: block;
+      height: 100%;
+      width: 3px;
+      position: absolute;
+      background: #00b9e4;
+      left: -2px;
+      top: 0;
+      z-index: 1;
     }
+    @media (min-width: @screen-sm-min) {
+      .flex(@columns: 0 0 49%);
+    } 
+    @media (min-width: @screen-md-min) {
+      .flex(@columns: 0 0 49.375%);
```

Comment:
If we have a value like this in more than one place, probably should be a variable.